src/cljs/org/broadinstitute/firecloud_ui/common/sign_in.cljs

```diff
@@ -71,9 +71,14 @@
                                   (let [token (get message "access_token")]
                                     (reset! u/access-token token)
                                     (u/set-access-token-cookie token)
-                                    (swap! state assoc :success? true)
-                                    (when-let [callback (:callback props)]
-                                      (callback))))}]]])]]))
+                                    (swap! state assoc :success? true)))}]]])]]))
+   :component-did-update
+   (fn [{:keys [state props]}]
+     (let [{:keys [success?]} @state]
+       (when success?
+         (modal/pop-modal)
+         (when-let [callback (:callback props)]
+           (callback)))))
```

Comment:
I'm not sure React would be okay with making UI changes during a render pass. This might be safer.

However, there is now an `if` statement on 63 that doesn't make sense anymore, along with the close button.